src/sysrepo.c

```diff
@@ -1574,6 +1574,11 @@ sr_install_module(sr_conn_ctx_t *conn, const char *schema_path, const char *sear
     return sr_install_module2(conn, schema_path, search_dirs, features, NULL, NULL, NULL, 0, NULL, NULL, 0);
 }
 
+API const sr_module_ds_t *
+sr_get_module_ds_default(void) {
```

Comment:
Opening bracket needs to be on a new line.

Reply:
Whoops, sorry about that. Tried to match the existing style wherever possible